README.md

```diff
@@ -1,2 +1,53 @@
 # APIv3-typescript-library
 SendinBlue's API v3 TypeScript Library
+
+SendinBlue's API exposes the entire SendinBlue features via a standardized programmatic interface. Please refer to the full [documentation](https://developers.sendinblue.com) to learn more.
+
+This is the wrapper for the API. It implements all the features of the API v3. It supports promises.
+
+SendinBlue's API matches the [OpenAPI v2 definition](https://www.openapis.org/). The specification can be downloaded [here](https://api.sendinblue.com/v3/swagger_definition.yml).
+
+## Installation
+
+### For [Node.js](https://nodejs.org/)
+
+#### npm
+
+The following recommended installation requires [npm](https://npmjs.org/). If you are unfamiliar with npm, see the [npm docs](https://npmjs.org/doc/).
+
+Then install it via:
+
+```shell
+npm install sib-api-typescript-node-v3-sdk --save
+```
+
+## Getting Started
+
+Please follow the [installation](#installation) instruction and execute the following sample code JS code for getAccount API:
+
+```javascript
+var SibApiV3Sdk = require('sib-api-typescript-node-v3-sdk');
+
+var api = new SibApiV3Sdk.AccountApi()
+
+// Configure API key authorization: api-key
```

Comment:
I suggest to rename _api-key_  -> _apiKey_  as it is being used in the code below on line 35  for consistency.